interpreter/cling/test/Extensions/Lookup/StillError.C

```diff
@@ -28,4 +28,10 @@ std::unique_ptr<cling::test::SymbolResolverCallback> SRC;
 SRC.reset(new cling::test::SymbolResolverCallback(gCling, false))
 gCling->setCallbacks(std::move(SRC));
 p.q // CHECK: {{input_line_.*: error: use of undeclared identifier 'p'}}
+
+// `auto` is not supported
+auto x = unknownexpression()
+// CHECK: {{input_line_[0-9]+:[0-9]+:[0-9]+}}: error: cannot deduce 'auto' from unknown expression
```

Comment:
What about adding `-verify` and `//expected-error`?

Reply:
If we have -verify, we would have to change that for p.q too, I tried to use something like
```
p.q // expected-error {{use of undeclared identifier 'p'}}

// `auto` is not supported
auto x = unknownexpression(); // expected-error {{cannot deduce 'auto' from unknown expression}}
```

But I get this (the string is the exact same, but the line number is different)
``
error: 'expected-error' diagnostics expected but not seen: 
  File input_line_27 Line 2: use of undeclared identifier 'p'
error: 'expected-error' diagnostics seen but not expected: 
  File input_line_28 Line 2: use of undeclared identifier 'p'
``

Reply:
Is there a combination that guarantees coverage and works? We need pragmatism... It would be nice to have this fixed asap :)

Reply:
As far as I understand, the file used `FileCheck` before. I agree it would be nice to use `-verify`, but I've seen other cases in the past where it was hard because of Cling line errors... Maybe this is one of the cases where you have to give the exact line number as `// expected-error@input_line_28 {{cannot deduce 'auto' from unknown expression}}` (and then sometimes suffer updating the line numbers when changing the test)